custom/enikshay/management/commands/base_model_reconciliation.py

```diff
@@ -61,3 +63,7 @@ def writerow(self, row):
         with open(self.result_file_name, 'a') as csvfile:
             writer = csv.DictWriter(csvfile, fieldnames=self.result_file_headers)
             writer.writerow(row)
+
+    @staticmethod
+    def get_first_opened_case(all_cases):
+        return sorted(all_cases, key=lambda x: x.opened_on)[0]
```

Comment:
nit: `min` would be more appropriate (faster too, although I doubt that's relevant here)
```python
return min(all_cases, key=lambda x: x.opened_on)
```